src/Scorecard.js

```diff
@@ -0,0 +1,64 @@
+class ScoreCard{
```

Comment:
Great succinct class - very clear what each method is doing and within the scope of the overall goals of the class!